src/ssl.c

```diff
@@ -11469,7 +11473,13 @@ int wolfSSL_SetServerID(WOLFSSL* ssl, const byte* id, int len, int newSession)
 
 #endif /* !NO_CLIENT_CACHE */
 
-#if defined(PERSIST_SESSION_CACHE)
+/* TODO: Add SESSION_CACHE_DYNAMIC_MEM support for PERSIST_SESSION_CACHE.
```

Comment:
Consider adding a settings.h rule to not allow `SESSION_CACHE_DYNAMIC_MEM` with `PERSIST_SESSION_CACHE`?
Please add `*` for comment section.